### ISSUE TITLE
Cep Não carrega!

### ISSUE THREAD

**Issue by MayaOliver** [deivisonarthur/OSC-Magento-Brasil-6-Pro#233]
Já tentei todos os tutoriais #177 #231 #200 #209  e não consigo arrumar... Só carrega eternamente e não sai disso... :(  Alguém pode me ajudar? Minha versão é a 1.9.2.2

Quando digito o CEP fica carregando e mostrando a mensagem "Por favor, corrija o CEP digitado, ele não está correto."

http://www.filtrosonline.com.br/

**Comment by rafaelstz** [comment]
Aparece algum erro no console? Verifica se no momento dos testes o webservice não estava fora.

**Comment by MayaOliver** [comment]
Olá Rafael, não aparece nenhum erro, testei o webservice e ele está funcionando normalmente...

**Comment by Codzen** [comment]
Mesmo problema está acontecendo para a gente.
Fica um carregamento sem fim na página, muitas vezes sendo necessário iniciar e finalizar o apache.

**Comment by rafaelstz** [comment]
Pode tentar em outro webservice, pode usar esse meu app para testar:
https://chrome.google.com/webstore/detail/c%C3%A1lculo-dos-correios/badfmkjhejbmeepnmiicdhmlabklioeg?utm_source=chrome-app-launcher-info-dialog

**Comment by Codzen** [comment]
Poisé, testando pelo seu app funcionou certinho, porém pelo one page checkout não, sendo que passei os mesmos parametros.

**Comment by MayaOliver** [comment]
Como posso inserir esse webservice no OSC? E fazer como que o OSC funcione corretamente?

**Comment by rafaelstz** [comment]
Nesse arquivo, linha 752.
https://github.com/deivisonarthur/OSC-Magento-Brasil-6-Pro/blob/master/app/code/community/Inovarti/Onestepcheckout/controllers/AjaxController.php

**Comment by Codzen** [comment]
Mas dai no lugar do https://apps.correios.com.br/SigepMasterJPA/AtendeClienteService/AtendeCliente?wsdl colocar o link da extensão igual tu mandou? Com "https://chrome.google.com/webstore/"??

**Comment by rafaelstz** [comment]
Não, você teria que mudar para outro link mesmo, mas de outro webservice, depois de mudar o link teria que mudar a maneira que ele passa os dados para esse link.

**Comment by MayaOliver** [comment]
Legal Rafael, você poderia exemplificar melhor?

**Comment by rafaelstz** [comment]
O desenvolvedor vai ter que tratar o código mas, para usar outro Webservice pode fazer essa chamada:

```php

$numero = $this->getRequest()->getParam('numero');
$numero = str_replace('-', '', $numero);

if (preg_match('/^\d{8}$/', $numero)) {
      $CEP_URL = @file_get_contents("http://cep.republicavirtual.com.br/web_cep.php?cep=".urlencode($numero)."&formato=json");
      echo $CEP_URL;
}

```

Estou com muita coisa pra fazer, caso contrário poderia explicar melhor, ou até mesmo criar um **path** git.
Espero que tenha ajudado :smile: :+1:

**Comment by thiagoromera** [comment]
Galera o meu fica assim. Vcs tentaram essa correção. Funcionou?
Veja o print: https://snag.gy/QuRLEF.jpg

**Comment by alexandrelrv99** [comment]
O meu também está da mesma forma thiagoromera, semana passada estava funcionando corretamente, agora nem carrega mais...

**Comment by racetech22** [comment]
Oi Pessoal,

Estava tendo o mesmo problema e, no meu caso, verifiquei que ocorreu depois que instalei o módulo para conectar o Zendesk.
Desinstalei este módulo e voltou a funcionar o campo CEP preenchendo os endereços automaticamente.

Acredito que possa ocorrer com outros módulos também.

Espero ajudar alguns com esta informação.

Abx,
Fernando
